.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
.idea folder is a resource folder Pycharm creates to be compatible with the project, it shouldn't be in the Github repository.
to add it, always have in your repository a .gitignore file (read about it)